Update README.md (#1397)
google/comprehensive-rust: README.md

- [mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers)
- [mdbook-exerciser](mdbook-exerciser/)

First install rust: 

```shell

> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh

Reviewer: I don't like to replicate these instructions — there should only be one place on the Internet that needs to know about the flags-of-the-day to use with `curl` :slightly_smiling_face: That is also why we refer to rustup.sh in the [Installation](https://google.github.io/comprehensive-rust/cargo.html#installation) section.

> First install rust: 
> 
> ```shell
> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
> source $HOME/.cargo/env
> rustc --version
> ```
> 
> Clone the repository:

Reviewer: Let us simply send people to rustup.rs:

````suggestion
First install Rust by following the instructions on https://rustup.rs/. Then clone this repository:
````

- [mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers)
- [mdbook-exerciser](mdbook-exerciser/)

> First install Rust by following the instructions on https://rustup.rs/. Then clone this repository:

Reviewer: Let me quickly fix the formatting according to `dprint`:

```suggestion
First install Rust by following the instructions on https://rustup.rs/. Then
clone this repository:
```
